internal/helpers.go

```diff
@@ -550,41 +551,41 @@ func FindStringInSlice(s []string, str string) bool {
 	return false
 }
 
-func ResetK1Dir(k1Dir string) error {
+func ResetK1Dir(k1Dir string, gitopsRepoName string, metaphorRepoName string) error {
 
 	if _, err := os.Stat(k1Dir + "/argo-workflows"); !os.IsNotExist(err) {
 		// path/to/whatever exists
 		err := os.RemoveAll(k1Dir + "/argo-workflows")
 		if err != nil {
-			return fmt.Errorf("unable to delete %q folder, error: %s", k1Dir+"/argo-workflows", err)
+			return fmt.Errorf("unable to delete %q folder, error: %w", k1Dir+"/argo-workflows", err)
 		}
 	}
 
-	if _, err := os.Stat(k1Dir + "/gitops"); !os.IsNotExist(err) {
-		err := os.RemoveAll(k1Dir + "/gitops")
+	if _, err := os.Stat(k1Dir + "/" + gitopsRepoName); !os.IsNotExist(err) {
```

Comment:
I saw you did it above, this is also a good candidate to use `filepath.Join()` instead, we would all appreciate it xD